x/perp/v2/module/genesis.go

```diff
@@ -91,5 +107,29 @@ func ExportGenesis(ctx sdk.Context, k keeper.Keeper) *types.GenesisState {
 		})
 	}
 
+	// export global discounts
+	discounts := k.GlobalDiscounts.Iterate(ctx, collections.Range[math.Int]{})
+	defer discounts.Close()
+	for ; discounts.Valid(); discounts.Next() {
+		genesis.GlobalDiscount = append(genesis.GlobalDiscount, types.GenesisState_Discount{
+			Fee:    discounts.Value(),
+			Volume: discounts.Key(),
+		})
+	}
+
+	// export custom discounts
+	customDiscounts := k.TraderDiscounts.Iterate(ctx, collections.PairRange[sdk.AccAddress, math.Int]{})
+	defer customDiscounts.Close()
+
```

Comment:
This file should have import and export tests to verify that we reproduce the same expected state from known inputs